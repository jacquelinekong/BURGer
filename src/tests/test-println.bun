println(3);
println("hellooo");
println("sooooooooo hmm");
println(true);
