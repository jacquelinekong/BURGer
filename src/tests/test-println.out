3
hellooo
sooooooooo hmm
1
